api/models/Post.js

```diff
@@ -0,0 +1,49 @@
+const db = require('../dbConfig/init')
+
+class Post {
+    constructor(data){
+        this.id = data.id,
+        this.title = data.title,
+        this.pname  = data.pname,
+        this.body = data.body;
+    };
+
+    static get all(){
+        return new Promise(async (resolve, reject) => {
+            try {
+                const post = await db.query(`SELECT * FROM posts`);
+                const posts = post.rows.map(r => ({ id: r.id, title: r.title, pname: r.pname, body: r.body}))
```

Comment:
instead of mapping out each id, title, etc the whole reason we make a class called Post is so we can just do:
`post.rows.map(r => new Post(r));` This makes it so the id and everything is what you have given those specific posts.